### DISCUSSION TITLE
Unable ungroup taskbar

### DISCUSSION THREAD

**Discussion by Ikif** [PeterStrick/ViVeTool-GUI#69] [General]
Hi.
I'm looking for option to disable grouping in taskbar.
I found in some site that I need to use ID 29785186, but it doesn't exist.
Thanks

**Comment by PeterStrick** [comment]
Try to manually use the ID by pressing F12 within ViVeTool GUI

**Reply by Ikif** [reply]
Hi.
I tried , I get " \Successfully set Feature ID 29785186 to Enable"
But in Personalization > Taskbar > Taskbar Behaviors the expected option doesn't appear.
win build : 22621.2134
Windows Feature Experience Pack 1000.22659.1000.0
ViVeTool_GUI ver. 1.7

**Reply by PeterStrick** [reply]
Ah I see. As per an article mentioning the ID, the ID only works for Windows Insiders in the Dev Channel. You are  running normal Windows. 
![image](https://github.com/PeterStrick/ViVeTool-GUI/assets/60312421/af3ad60f-c486-4687-b10e-0ba997424a0e)

Source: https://www.xda-developers.com/how-enable-taskbar-ungrouping-labels-windows-11/
